src/components/templates/descopeMiddleware.tsx

```diff
@@ -130,6 +130,24 @@ export const DescopeMiddleware = ({ children }: { children: ReactNode }) => {
 		setDescopeRenderKey((prevKey) => prevKey + 1);
 	};
 
+	const handleCliLoginRedirect = (): boolean => {
+		const redirPath = Cookies.get(systemCookies.redir);
+
+		if (redirPath) {
+			Cookies.remove(systemCookies.redir, { path: "/", domain: ".autokitteh.cloud" });
+
+			if (redirPath.startsWith("/auth/")) {
+				const apiBaseUrl = getApiBaseUrl();
+				const backendUrl = `${apiBaseUrl}${redirPath}`;
+
+				window.location.href = backendUrl;
+				return true;
```

Comment:
IMPORTANT: You should return `true` after changing the `window.location.href` to `backendUrl`. This implies that the page redirects to the `backendUrl` and subsequently returns true, which is not entirely logical. Please clarify this solution or propose an alternative.